kani-compiler/src/kani_compiler.rs

```diff
@@ -364,3 +380,75 @@ impl Callbacks for KaniCompiler {
         self.prepare_codegen()
     }
 }
+
+#[cfg(test)]
+mod tests {
```

Comment:
Love this! Is there any tool the measure test coverage that we should be using in CI?

Reply:
There are tools out there that we could and should be using. Unfortunately, we haven't integrated any yet. I think it's important for us to collect data on end-to-end tests, since most of our testing is done at that level.

Reply:
Do you mean "end-to-end" test by our current regression tests?